pwnagotchi/locale/de/LC_MESSAGES/voice.po

```diff
@@ -62,6 +69,13 @@ msgstr "Ich bin sehr traurig..."
 msgid "I'm sad"
 msgstr "Ich bin traurig"
 
+#, fuzzy
+msgid "Leave me alone ..."
+msgstr "Lass mich in Ruhe..."
```

Comment:
Is "Lass mich allein ..." not a more appropiate translation for this?

Reply:
"Verpiss Dich!" might be a bit more aggressive?